src/displayapp/screens/settings/SettingWatchFace.cpp

```diff
@@ -81,4 +90,4 @@ void SettingWatchFace::UpdateSelected(lv_obj_t* object, lv_event_t event) {
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Unnecessary file ending change

Reply:
It's adding a newline at the end of the file - some files have them, some don't, it seemed easier to add them when they're missing than to strip them out everywhere.

Reply:
I think this is not currently enforced as a standard, the next person will revert again and so on and on before tooling allows a easy way to enforce. So it's better not to change.